rules/tournaments/aoe2de.md

```diff
@@ -0,0 +1,128 @@
+# Age of Empires 2
+
+## General Provisions
+
+These rules inherit all rules from the main rulebook.
+
+These rules apply to all participants of the "Age of Empires 2" tournament (hereinafter "AoE2" or "Game").
+
+The rules may be available in several languages. If the individual versions differ, the English version is considered the source of the truth.
+
+## Tournament Information
+
+### Tournament Platform
+
+The official tournament platform for this tournament is Battlefy.
+Please make sure that you have an account: https://battlefy.com/account/signup
+
+The official tournament page will be shared via the main communication channel established in the main rulebook.
+
+### Tournament Format
+
+The tournament is played in the following stages:
+
+| Stage   | Type               | Format    | Description                                                            |
+|---------|--------------------|-----------|------------------------------------------------------------------------|
+| Stage 1 | Swiss Rounds       | Bo1       | This stage will be seeded according to current 1v1 RM ELO rankings.    |
+| Stage 2 | Single Elimination | Bo1 / Bo3 | The bracket will be seeded according to the result of the first stage. |
+
+### Qualification Swiss System
+
+4 rounds are played according to the Swiss-System. The games will be reallocated after each round according to the
+current ranking. For the ranking the won and lost games count. The best eight players will then play for the tournament
+victory. If there is a tie for 8th place after the 4 rounds, the ranking will be calculated automatically.
+
+## Schedule
+
+The first appointment for the team representative will be the briefing on Saturday at 15:30.
+The tournament itself will start one hour later at 16:30.
+All players must be at the location at the start of the tournament.
+
+No accommodations will be made for other tournaments that may happen during the time of this tournament.
```

Comment:
This line is only valid for T1 tournaments, Other tiers should accommodate those.